### PR TITLE
Made naquadah reactor use bolt as fuel

### PR DESCRIPTION
**Pull request by IamNotBrianZ**
NomiCEu parity

### CODE REVIEW

**Comment by WithersChat** [comment]
I checked the changes, it's all in order.

**Comment by IamNotBrianZ** [comment]
Wow, this is almost a month ago

**Comment by WithersChat** [comment]
I know. I commented so Pansmith cn just click one button and merge it.

File: kubejs/server_scripts/gregtech/naquadah_reactor.js [hunk 1]
```diff
@@ -6,40 +6,40 @@ ServerEvents.recipes(event => {
...
+        .stationResearch(b => b.researchStack(Item.of('gtceu:naquadah_bolt')).EUt(30720).CWUt(16,64000))
 
     event.recipes.gtceu.assembly_line('kubejs:naquadah_reactor_ii')
         .itemInputs('gtceu:computer_monitor_cover', '3x ae2:spatial_pylon', '#gtceu:circuits/uv', '2x gtceu:fusion_glass')
         .inputFluids('gtceu:soldering_alloy 1152', 'gtceu:omnium 288')
         .itemOutputs('gtceu:naquadah_reactor_ii')
         .duration(1500)
         .EUt(122880)
-        .stationResearch(b => b.researchStack(Item.of('gtceu:naquadah_reactor_i')).EUt(131072).CWUt(2048))
+        .stationResearch(b => b.researchStack(Item.of('gtceu:naquadah_reactor_i')).EUt(122880).CWUt(64,256000))
```
Comment (ThePansmith) [inline kubejs/server_scripts/gregtech/naquadah_reactor.js:17]: revert this, no reason to have cwut to be this high

Comment (IamNotBrianZ) [reply kubejs/server_scripts/gregtech/naquadah_reactor.js:17]: wdym, I remember cwut(2048) means it requires 2048 cwut
I actually made cwut lower to 64

Comment (IamNotBrianZ) [reply kubejs/server_scripts/gregtech/naquadah_reactor.js:17]: 螢幕擷取畫面 2024-07-16 171123
This is what "CWUt(2048)" looks like, and it is impossible to reach 
So i change it to the value in NomiCEU

**Pull request merged**